api/v1alpha1/agent/openapi.yaml

```diff
@@ -0,0 +1,182 @@
+openapi: 3.0.1
+info:
+  title: Open Device Management API - agent endpoint
+  version: undefined
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0.html
+servers:
+  - url: /
+paths:
+  /api/v1/devices/{name}/status:
+    put:
+      tags:
+        - device
+      description: replace status of the specified Device
+      operationId: replaceDeviceStatus
+      parameters:
+        - name: name
+          in: path
+          description: name of the Device
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '../openapi.yaml#/components/schemas/Device'
+        required: true
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '../openapi.yaml#/components/schemas/Device'
+        "400":
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '../openapi.yaml#/components/schemas/Error'
+        "401":
+          description: Unauthorized
+          content:
+            application/json:
+              schema:
+                $ref: '../openapi.yaml#/components/schemas/Error'
+        "404":
+          description: NotFound
+          content:
+            application/json:
+              schema:
+                $ref: '../openapi.yaml#/components/schemas/Error'
+  /api/v1/devices/{name}/rendered:
```

Comment:
```
  /api/v1/devices/{name}/rendered:
    $ref: '../openapi.yaml#/paths/~1api~1v1~1devices~1{name}~1rendered'
  /api/v1/enrollmentrequests/{name}:
    $ref: '../openapi.yaml#/paths/~1api~1v1~1enrollmentrequests~1{name}'
```

To avoid duplication we need something like this, but it seems to generate buggy code in oapi-gen v1.x , v2 seems to fix it, checking if it's easy to update to v2.